jsapp/js/components/common/koboDropdown.es6

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import autoBind from 'react-autobind';
+import {bem} from 'js/bem';
+import {KEY_CODES} from 'js/constants';
+import './koboDropdown.scss';
+
+export const KOBO_DROPDOWN_THEMES = {};
+new Set([
```

Comment:
Should we be declaring all local constant sets this way? I've usually just been doing something like `set = [one: {id:'one'}, ...]`

Reply:
I like this way because you don't risk a typo in the name - in `one: {id: 'one'}` you need "one" and "one" to be identical, and althought it seems pretty simple there is some risk of a bug. With this convoluted way there is none :) Plus the freeze is always nice :D